src/devices/Pcd8544/NokiaFont.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// The specific PCD8544 font for Nokia 5110
+    /// </summary>
+    public static class NokiaCharacters
+    {
+        /// <summary>
+        /// ASCII Font specific to the PCD8544 Nokia 5110 screen.
+        /// Font characters are column bit mask.
+        /// Font size is 5 pixels width and 8 pixels height. Each byte represent a vertical column for the character.
+        /// </summary>
+        public static readonly byte[][] Ascii = new byte[][]
```

Comment:
note that only the outer array is readonly but you can still edit the content. I don't expect any action here as I can imagine users might want that.

Reply:
yes. But even if a developer want to change a specific part of the font, that's kind of acceptable if they know what they are doing!